src/config.rs

```diff
@@ -639,9 +639,15 @@ make_config! {
         /// Timeout when acquiring database connection
         database_timeout:       u64,    false,  def,    30;
 
-        /// Database connection pool size
+        /// Timeout in seconds before idle connections to the database are closed
+        database_idle_timeout:  u64,    false, def,     60;
```

Comment:
```suggestion
        database_idle_timeout:  u64,    false, def,     600;
```

The default value of this in `r2d2` is 10 minutes (600 seconds), so changing it to be now 60 seems unnecessary here. We can let it be configurable but follow the default provided to us.

https://docs.rs/r2d2/latest/r2d2/struct.Builder.html#method.idle_timeout

Reply:
No problem, the original value was arbitrary.